lib/react.dart

```diff
@@ -5,14 +5,28 @@
 /// A Dart library for building UI using ReactJS.
 library react;
 
+import 'dart:html' show Element;
```

Comment:
`lib/react.dart` is also used for server-side rendering, and should not import `dart:html`